src/api/services/participantsService.ts

```diff
@@ -67,10 +73,20 @@ export const getParticipantsAwaitingApproval = async (email: string): Promise<Pa
         .select('participantId')
     )
     .withGraphFetched('types')
+    .withGraphFetched('users')
```

Comment:
nit: I think we could do `.withGraphFetched('[types, users]')`, maybe they have performance as chain it up

Reply:
Done.